packages/ingest/readers/unifi.py

```
@@ -253,164 +275,542 @@ def _fetch_clients(self) -> list[dict[str, Any]]:
+                        gateway_ip=None,  # Not available in site API
+                        dns_servers=None,  # Not available in site API
+                        created_at=now,
+                        updated_at=now,
+                        source_timestamp=None,
+                        extraction_tier="A",
+                        extraction_method="unifi_api",
+                        confidence=1.0,
+                        extractor_version=EXTRACTOR_VERSION,
+                    )
+                    sites.append(site)
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)